### PR TITLE
Import lost donations into DB cronjob 

### PR DESCRIPTION
**Pull request by CarlosQ96**
Related to: https://github.com/Giveth/giveth-dapps-v2/issues/3526

### CODE REVIEW

**Comment by CarlosQ96** [comment]
Will not import multisig donations, those were not part of the round and we just released that. There should not be any or any communication about it. Not sure how they were done.

File: src/services/cronJobs/importLostDonationsJob.ts [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+    transaction.blockNumber as number,
+  );
+
+  const transactionTo = transaction.to;
+  const amount = ethers.utils.formatEther(transaction.value);
+
+  return {
+    from: transaction.from,
+    timestamp: block.timestamp as number,
+    to: transactionTo as string,
```
Comment (mateodaza) [inline src/services/cronJobs/importLostDonationsJob.ts:237]: Awesome job man. @CarlosQ96 
Just make sure you're checking the from and to values on the Transfer event from the logs instead. Then you're making sure you're grabbing the right values each time. Check the multisig code for it, we did something similar

**Review by mateodaza — APPROVED** [review]

File: src/services/cronJobs/importLostDonationsJob.ts (hunk 1 above)
Comment (CarlosQ96) [reply src/services/cronJobs/importLostDonationsJob.ts:237]: Thanks man

**Comment by CarlosQ96** [comment]
@aminlatifi @mohammadranjbarz can review please so we import the donations. Ill handle the optimism ones first then the rest. This should work for all.

**Comment by mohammadranjbarz** [comment]
> @aminlatifi @mohammadranjbarz can review please so we import the donations. Ill handle the optimism ones first then the rest. This should work for all.
@CarlosQ96 
Name PR should not be 
**Import lost donations into DB cronjob** ?

File: src/services/cronJobs/importLostDonationsJob.ts [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+        givbackFactor,
+        projectRank,
+        powerRound,
+        bottomRankInRound,
+        status: 'verified',
+        anonymous: false,
+        segmentNotified: true,
+        isTokenEligibleForGivback: tokenInDB?.isGivbackEligible,
+        isProjectVerified: project.verified,
+        qfRoundId: qfRound?.id,
```
Comment (mohammadranjbarz) [inline src/services/cronJobs/importLostDonationsJob.ts:197]: @CarlosQ96  Should not add qfRound to all missed donations, we should check if project is connected to qfRound, then add `qfRoundId` to donation ( and its nice to have a test case for that scenario as well)

**Review by mohammadranjbarz — CHANGES_REQUESTED** [review]

File: src/services/cronJobs/importLostDonationsJob.ts (hunk 2 above)
Comment (mohammadranjbarz) [reply src/services/cronJobs/importLostDonationsJob.ts:197]: I did it myself just please take a look yourself too

**Review by mohammadranjbarz — APPROVED** [review]
Thanks @CarlosQ96 
I skipped the changes for qfRound as it's just for our tx hashes

**Pull request merged**